frontend/src/app/libs/landing/components/public-events-list/public-events-list.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Event } from '../../models/event';
+import { EventsFacadeService } from '../../store/events/events.facade';
+
+@Component({
+	selector: 'app-public-events-list',
+	templateUrl: './public-events-list.component.html',
+	styleUrls: ['./public-events-list.component.scss'],
+})
+export class PublicEventsListComponent {
+	constructor(private eventsFacadeService: EventsFacadeService) {}
+
+	public get eventsValue$(): Observable<Event[]> {
```

Comment:
no need to copy originam method name, since in this component we have `events` context so lets keep `events$():...` name